source/magpie/dataframe.d

```diff
@@ -1220,6 +1220,44 @@ public:
             return ret;
         }
     }
+
+    /++
+    auto columnToIndex(int position)() @property
+    Description: Converts a colimn into row index level
```

Comment:
col_i_mn should be col_u_mn

Reply:
Nice catch!
I wouldn't have spotted this in a million years with my editor theme :sweat_smile: 
Addressed in dec86d1